scripts/trim-low-abund.py

```diff
@@ -63,18 +63,9 @@
                          check_valid_file_exists, add_output_compression_type,
                          get_file_writer)
 
-DEFAULT_NORMALIZE_LIMIT = 20
+DEFAULT_TRIM_AT_COVERAGE = 20
 DEFAULT_CUTOFF = 2
```

Comment:
Just a thought, but considering the emphasis we put on our our default settings, it might be valuable to keep these in their own file in the main lib, like `defaults.py` or some such, where they can be imported and interrogated.